php/libraries/Candidate.class.inc

```diff
@@ -638,8 +638,35 @@ class Candidate implements \LORIS\StudyEntities\AccessibleResource
 
         return $subprojList;
     }
+
+    /**
+     * Returns the SubprojectID and subproject title of the candidate's last visit
+     * based on Date_visit
+     *
+     * @return array|null Array with values SubprojectID and title
+     *                    for the candidate's last visit or NULL if none exist
+     */
+    public function getSubprojectForMostRecentVisit(): ?array
```

Comment:
Remind me why you are creating this function but not using it anywhere ?? (it's completely removed from the instrument_list module)

Reply:
i use it in QPN's candidate_profile page to determine which subproject the participant belongs to given their lastest visit. it replaces the Utility::getSubprojectIDUsingCandID function

Reply:
That answers my question

Reply:
then why doesn't the function live in the QPN's candidate_profile page if that's the only place it's used?

Reply:
wouldnt it be used for the new LORIS candidate profile too ?
i thought it was based on QPN's

Reply:
no, it's not used by the new LORIS `candidate_profile`